### ISSUE TITLE
Issue with Win8

### ISSUE THREAD

**Issue by Darlingtone** [lepipele/csharpWebCam#3]
Hi, I used this library in my WPF application and it worked great on my Acer laptop running Win8.1, but on a fresh out of the box HP ProOne 400 system running Win8.1 Pro the App will crash when I try bringing up the capture window.

The stack trace shows that the error occurred when WebCamLib tries to Enumerate the Available webcam

Any ideal as to why or solution?

Thanks

Darlington

P.S. I allowed VS to upgrade the WebCamLib to VS 2013 compatibility; cause i don't have and don't want to install VS-2010 Express - more so when i don't plan to change the code or contribute back to the repo. And the HP Win8.1 is not yet fully updated to the current win8.1 compare to my development laptop

**Comment by lepipele** [comment]
Well, I'm still on Windows 7 - so I could only speculate what is going on.

If you do find solution, please leave comment here to help others if they run into similar problem.

**Comment by jayesh14** [comment]
Hello, I am facing same issue as per Darlington. 
What is the dependence software i need to install for this lib?  
I have installed visual c++ 2010 redistributable x86 software(10.0.40219) but i am facing this issue.
And Also this demo is not working on that PC.
Any solution?
Thanks,

**Comment by lepipele** [comment]
As I said to @Darlingtone - I'm sorry that I can't test on Win 8.1.

@jayesh14 you can also try posting directly on article forum that's on:
http://www.codeproject.com/Articles/125478/Versatile-WebCam-C-library#_comments

Dunno if @FluxIX (who contributed pull request) has any idea on this can be resolved... I'll keep issue opened in case somebody who has solved the problem comes upon this topic.

**Comment by FluxIX** [comment]
If I had to guess, the camera supports a different resolution than the library supports when it tries to take the image from the camera and create a bitmap which can be sent through the event. If @Darlingtone or @jayesh14 is willing to test this theory (and has a debugger to test with), walk through the CameraMethods::StartCamera in WebCamLib.cpp and check what pixel format is being selected; the pixel type should be either RGB (24 bits) or ARGB (32 bits, although the alpha channel ignored and set to 0xFF).

**Comment by vikramjb** [comment]
I am on Win 8.1 and the capture worked pretty good. I however have the VS 2010 pro version on my machine. Can you post your stack trace, I can take a look where it is crashing.

**Comment by jayesh14** [comment]
Hello Vikramjb
I thinks, This is not issues with OS 8.1 and this code. We have used c++ code(made DLL by 32 bit) then we need to installed related software i means dependence software. 
I am writing when i was facing this issue.
I had tested this code with many PC and difference OS.
I was test with new fresh PC.
List of installed software 
1) OS 8.1 64 bit
2) Framework 4.5
3) Microsoft Visual C++ 2010 SP1 Redistributable Package (x86)

I had tried with above configuration. without installed any visual studio 
That's why we are facing this issue.

**Comment by Darlingtone** [comment]
Like I said, it worked on my Win 8.1 development machine and a fresh Win 8, but on one particular Win 8.1 ( HP All in One ) Machine it always crash when you start the WebCam








Sent from Windows Mail





From: Vikram Ramanthan
Sent: ‎Friday‎, ‎July‎ ‎24‎, ‎2015 ‎5‎:‎04‎ ‎AM
To: lepipele/csharpWebCam
Cc: Darlington ADIBE





I am on Win 8.1 and the capture worked pretty good. I however have the VS 2010 pro version on my machine. Can you post your stack trace, I can take a look where it is crashing.

**Comment by vikramjb** [comment]
Hi Jayesh

So you are staying the DLL crashes when you use the compiled version of the
application. Is it ?

**Comment by jayesh14** [comment]
Yes, Crash application Because  dependence dll/application/software are missing.  
I thinks, code is okay.

I have just copied exe file from debug(or release) one PC (development PC(Installed Visual studio)) . Just paste this exe file on other PC(client PC(Not installed Visual studio related software)).
So, need to find dependence software for c++ dll with OS 32 bit and 64 bit.

**Comment by lepipele** [comment]
@jayesh14 See if this helps: 

http://www.codeproject.com/Articles/125478/Versatile-WebCam-C-library?msg=5116145#xx5116145xx
